implement/applications/eve-online/eve-online-mining-bot/Bot.elm

```diff
@@ -3,10 +3,14 @@
    The bot warps to an asteroid belt, mines there until the mining hold is full, and then docks at a station or structure to unload the ore. It then repeats this cycle until you stop it.
    If no station name or structure name is given with the bot-settings, the bot docks again at the station where it was last docked.
 
-   Setup instructions for the EVE Online client:
+   ## Setting up the Game Client
+
+   Despite being quite robust, this bot is less intelligent than a human. For example, his perception is more limited than ours, so we need to configure the game to make sure the bot sees everything it needs to, and we're going to hide some things it doesn't need, here's a list of configuration instructions for the EVE client Online:
 
    + Set the UI language to English.
-   + In the ship UI in the 'Options' menu, tick the checkbox for 'Display Module Tooltips'.
+   + In the ship UI in the 'Options' menu:
+     + Hide passive modules by disabling the checkbox 'Display Passive Modules'.
```

Comment:
As far as I understand it, this one is optional, right?

Since it is placed above the required points, people who read this will need more time to arrive at the more important points for the setup. If something is not working and they don't understand that new point, how do they know that this is not the cause for the problems they are seeing?

Reply:
> As far as I understand it, this one is optional, right?
> 
> Since it is placed above the required points, people who read this will need more time to arrive at the more important points for the setup. If something is not working and they don't understand that new point, how do they know that this is not the cause for the problems they are seeing?


Yes, and an option or recommendation, I agree with you, it makes perfect sense.

Can we then create a recommendation section for the game client?

- Hide passive modules by disabling the 'Display Passive Modules' checkbox.
- set the Transparency to 0 (Zero) (move the transparency slider all the way to the left)
- Create an new windows overview with only warp to dock